MediaGalleryRenditions/Test/Integration/Model/GetRenditionPathTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryRenditions\Test\Integration\Model;
+
+use Magento\Framework\App\Filesystem\DirectoryList;
+use Magento\Framework\Filesystem;
+use Magento\Framework\Filesystem\Directory\WriteInterface;
+use Magento\Framework\Filesystem\DriverInterface;
+use Magento\MediaGalleryRenditionsApi\Api\GetRenditionPathInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+class GetRenditionPathTest extends TestCase
+{
+
+    /**
+     * @var GetRenditionPathInterface
+     */
+    private $getRenditionPath;
+
+    /**
+     * @var WriteInterface
+     */
+    private $mediaDirectory;
+
+    /**
+     * @var DriverInterface
+     */
+    private $driver;
+
+    protected function setup(): void
+    {
+        $this->getRenditionPath = Bootstrap::getObjectManager()->get(GetRenditionPathInterface::class);
+        $this->mediaDirectory = Bootstrap::getObjectManager()->get(Filesystem::class)
+            ->getDirectoryWrite(DirectoryList::MEDIA);
+        $this->driver = Bootstrap::getObjectManager()->get(DriverInterface::class);
+    }
+
+    /**
+     * @dataProvider getImageProvider
+     *
+     * Test for getting a rendition path.
+     */
+    public function testExecute(string $path, string $expectedRenditionPath): void
+    {
+        $imagePath = realpath(__DIR__ . '/../../_files' . $path);
+        $modifiableFilePath = $this->mediaDirectory->getAbsolutePath($path);
+        $this->driver->copy(
+            $imagePath,
+            $modifiableFilePath
+        );
+        $getRenditionPath = $this->getRenditionPath->execute($path);
+        $this->assertEquals($expectedRenditionPath, $getRenditionPath);
```

Comment:
Unnecessary variable
```suggestion
        $this->assertEquals($expectedRenditionPath, $this->getRenditionPath->execute($path));
```